data/class_whitelist.json

```diff
@@ -1,4 +1,5 @@
 [
+  "FObject",
```

Comment:
2 `FObject`s?

Reply:
I've learned that the default package resolution for whitelist contexts is sadly brittle:

Looking up `Foo` defaults to looking up `foam.core.Foo`. Fine. Reasonable. But the addition of the default package happens after the whitelist context performs its checks, so it's possible to have code `lookup('FObject')` in one place and `lookup('foam.core.FObject')` in another. They will resolve to the same class, but both need to be whitelisted :cry:

Reply:
Maybe add a comment indicating why there are duplicates, so that a future editor has a chance of doing the thing that is required to make it work.

Reply:
Sadly, I don't think comments are a part of the JSON spec. Maybe there should be a `data/README.md`?